src/main/java/com/epam/ta/reportportal/dao/TestItemRepositoryCustom.java

```diff
@@ -283,6 +284,14 @@
 	 */
 	Optional<IssueType> selectIssueTypeByLocator(Long projectId, String locator);
 
+	/**
+	 * Select id and path for item by uuid
+	 *
+	 * @param uuid {@link TestItem#getUuid()} ()}
+	 * @return id from collection -> {@link PathName}
+	 */
+	Optional<Pair<Long, String>> selectPathName(String uuid);
```

Comment:
actually it isn't a path name. it is just a path. maybe it is better to rename the method?

Reply:
renamed